android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -102,8 +103,26 @@ class RNPlayerView(
     private val activityLifecycle = (context.currentActivity as? ReactActivity)?.lifecycle
         ?: error("Trying to create an instance of ${this::class.simpleName} while not attached to a ReactActivity")
 
+    /**
+     * Relays the provided set of events, emitted by the player, together with the associated name
+     * to the `eventOutput` callback.
+     */
+    private var playerEventRelay: EventRelay<Player, Event>
+
+    private var playerInMediaSessionService: Player?
+        get() = context.playerModule?.mediaSessionPlaybackManager?.serviceBinder?.player
+        set(value) {
+            value?.let {
+                context.playerModule?.mediaSessionPlaybackManager?.serviceBinder?.player = it
+            }
+        }
+
     private val activityLifecycleObserver = object : DefaultLifecycleObserver {
+        // Don't stop the player when going to background
         override fun onStart(owner: LifecycleOwner) {
+            if (playerInMediaSessionService != null) {
+                player = playerInMediaSessionService
```

Comment:
Does this mean that if the media session is active and then the user opens up another video with a different player configuration it will be ignored right? 

So this scenario: 
- App has a list of video items (all have background playback enabled)
- User opens the first video and we create a media session
- User goes to background (where the app might be killed)
- Player persists in the media session
- User opens the app and navigates to a different video item
- Instead of playing the other video the player instance will be overriten by the player in the media session

I'm just trying to understand if this is a known limitation that we are ok with 🙂

Reply:
Address with the changes to state-management in 36e88e6d28a8066797fef76bb963fb2dc930c50b